Bottom: 319645469d8beef84bdea870c2a851a531f692b8
Top:    e1fa64b7f4a54d474fe94c634bb3cc44ad1004b2
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-11 11:16:00 +0800

Refresh of net-core-factor-out-core-xdp

---

diff --git a/net/core/dev.c b/net/core/dev.c
index b2c3ba1..9babea7 100644
--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -4230,7 +4230,6 @@ static u32 do_xdp_generic_core(struct sk_buff *skb,
 	return act;
 }
 
-
 static u32 netif_receive_generic_xdp(struct sk_buff *skb,
 				     struct xdp_buff *xdp,
 				     struct bpf_prog *xdp_prog)
